packages/match/configure/src/answer-config-block.jsx

```diff
@@ -21,7 +21,7 @@ const styles = (theme) => ({
     display: 'flex',
     alignItems: 'center',
     flex: 1,
-    width: 'fit-content',
+    width: '100%',
```

Comment:
Can you please check with @andreeimiron if this change doesn't affect his work?
<img width="1263" alt="Screenshot 2023-10-12 at 11 04 38" src="https://github.com/pie-framework/pie-elements/assets/20281464/d685fea4-0ad9-4549-b706-730406701def">

Reply:
@andreeapescar, @andreeimiron please take a look at my Issue and Fix Description for Row Misalignment provided in the ticket https://illuminate.atlassian.net/browse/PD-3231?focusedCommentId=1224523